project/Dependencies.scala

```diff
@@ -29,22 +29,7 @@ object Dependencies {
   private val utilScripted = "org.scala-sbt" %% "util-scripted" % utilVersion
 
   private val libraryManagementCore = "org.scala-sbt" %% "librarymanagement-core" % lmVersion
-
```

Comment:
why this configuration values got removed?
if this is the case remember that the integration tests in lm will stop to work(e.g. they should be removed as well).

Reply:
I figured that if we adopted lm-coursier from coursier repo, we can remove one from sbt/librarymanagement.